### ISSUE TITLE
Support SQLite OR provide docker-compose.yml for it

### ISSUE THREAD

**Issue by gardiol** [SamR1/FitTrackee#656]
I understand that PGSQL is the current only DB supported. 

I ask that you reconsider your choice as depoying a full set PGSQL is cumbersome for single user installs where SQLite is a much better approach.

If you provided a docker-compose installation approach with built-in PGSQL image probably this issue would be less severe, but at this time i decided to stop using FitTrackee after i decided i do not want to manage an additional bare-metal database engine just for this tool.

TLDR; to ensure broader adoptance of this great tool, either support SQLite or provide a good to go docker-compose.yml file.

(feel free to close this issue right away, it's your project, just please reconsider these choices)

**Comment by SamR1** [comment]
Hi,

No problem, I understand (for my part, I've sometimes given up installing applications using MySQL because I didn't want to have a database system other than PostgreSQL or SQLite).

The question of another database system has already been asked. As I plan to use PostgreSQL extensions when rewriting workouts processing, I have no plans to integrate another system at the moment. I may reconsider if SQLite provides the same functionality and its integration doesn't involve too much adaptation at that time.

About docker, I only use it for development (I don't really have any experience of managing applications using docker in production).  
However, I have provided an example of `docker-compose.yml` in this [repository](https://github.com/SamR1/docker-fittrackee), which could certainly be updated or improved.

**Comment by DavidHenryThoreau** [comment]
I've built image for FitTrackee and you can use that [docker-compose.yaml](https://github.com/DavidHenryThoreau/FitTrackee/blob/master/docker-compose.yaml)

**Comment by gardiol** [comment]
Wow, thank you! This is amazing... Maybe @SamR1 can add this compose in the repo?
One more question, will this compose be compatible with the garmin integration as well?

**Comment by DavidHenryThoreau** [comment]
I've made a pull request https://github.com/SamR1/FitTrackee/pull/654
Which garmin integration are you talking about ?

**Comment by SamR1** [comment]
Docker images are now available on [Docker Hub](https://hub.docker.com/r/fittrackee/fittrackee/tags) and [GitHub container registry](https://github.com/SamR1/FitTrackee/pkgs/container/fittrackee). 

Link to documentation: https://samr1.github.io/FitTrackee/en/installation.html#id6

Thank @DavidHenryThoreau for initiating the pull request.
